examples/demo/xtask/src/main.rs

```diff
@@ -50,14 +47,14 @@ fn main() -> Result<()> {
             }
         }
         Command::Watch(arg) => {
-            log::info!("Starting to watch");
+            log::info!("Watching for changes and check...");
+            let mut build_command = process::Command::new("cargo");
+            build_command.arg("check");
             arg.run(build_command)?;
         }
-        Command::Serve(arg) => {
-            log::info!("Starting to serve");
-            build_command.arg("--optimize");
-            arg.command(build_command)
-                .start(xtask_wasm::default_dist_dir(false))?;
+        Command::Start(arg) => {
+            log::info!("Starting to development server...");
+            arg.arg("dist").start(xtask_wasm::default_dist_dir(false))?;
```

Comment:
Can you use this in the `DevServer` example too?

src/dev_server.rs line 43-49

Reply:
Good catch!